tracker_scraper.go

```diff
@@ -74,6 +74,7 @@ func (me *trackerScraper) announce() (ret trackerAnnounceResult) {
 		ret.Completed = time.Now()
 	}()
 	ret.Interval = 5 * time.Minute
+	ret.Err = nil
```

Comment:
Lets pretend we want to announce second time. Still want to keep pervious error?